### DISCUSSION TITLE
Display .bit name & avatar

### DISCUSSION THREAD

**Discussion by aliez-ren** [wevm/wagmi#1438] [Ideas]
https://www.did.id/

> .bit is a blockchain-based, open source, decentralized cross-chain account system that provides a worldwide unique naming system with a .bit suffix that can be used in different scenarios, such as cryptocurrency transfer, domain name resolution, identity authentication, etc.

**Comment by 457813723** [comment]
Good idea. 
[.bit](https://www.did.id/)
